src/current/_includes/cockroachcloud/backups/provision-cluster-tf-managed-backups.md

```diff
@@ -0,0 +1,16 @@
+You may also want to configure the [managed backup]({% link cockroachcloud/managed-backups.md %}) settings your CockroachDB {{ site.data.products.standard }} cluster takes automatically. To do so, include the `backup_config` attribute in the `cockroach_cluster` resource:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ hcl
+    backup_config = {
+        enabled           = true
+        frequency_minutes = 60
+        retention_days    = 30
+    }
+    ~~~
+
+    {{site.data.alerts.callout_info}}
+    You can modify the [retention]({% link cockroachcloud/managed-backups.md %}#retention) of managed backups only once with either the [Cloud Console]({% link cockroachcloud/managed-backups.md %}#cloud-console), the [Cloud API]({% link cockroachcloud/managed-backups.md %}#cloud-api), or Terraform.
```

Comment:
should we mention here that further edits of retention can be accomplished by reaching out to support?

Reply:
edit: I see its mentioned on the other include.  If the include is nearby this include, it may not be necessary.

Reply:
I added the support mention into this include too, because it is in a spot in the docs. Good call!